supplier_app/models.py

```diff
@@ -401,6 +406,27 @@ def __str__(self):
         )
 
 
+class BankAccountArgentina(models.Model):
+    payment_type = models.IntegerField(
+        choices=get_payment_type_info_choices(),
+        verbose_name=_('Payment type')
+    )
+    account_type = models.IntegerField(
+        choices=get_account_type_info_choices(),
+        verbose_name=_('Account type')
+    )
+    beneficiary = models.CharField(
+        max_length=60,
+        verbose_name=_("Beneficiary"),
+    )
+    bank_account = models.ForeignKey(
```

Comment:
I think that this relation should be a OneToOne, or is there any reason on having multiple Argentinian bank accounts for one general bank account.